client/js/components/procedure/admin/AuthorizedUsersList.vue

```diff
@@ -102,10 +102,9 @@
           aria-hidden="true" />
         {{ Translator.trans('export') }}
       </a>
-      <i
-        class="fa fa-question-circle inline-block align-top u-ml-0_25 u-mt-0_125"
-        :aria-label="Translator.trans('contextual.help')"
-        v-tooltip="Translator.trans('consultation.export.bulk.letter.explanation')" />
+      <dp-contextual-help
+        class="inline-block align-top u-ml-0_25 u-mt-0_125"
```

Comment:
I can imagine that `align-top` is no longer needed.

Reply:
ok, i see this component is not in use within any project at the moment. in this case we may leave it as it is.

Reply:
@spiess-demos 
`DpSurveyStatus` or `AuthorizedUsersList`

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2955/commits/b2a9d5597cbcb27fb2155c1cee68774d3fe5d0a6 for `AuthorizedUsersList`

Reply:
DpSurveyStatus is not used at the moment i guess.